manager/dispatcher/dispatcher.go

```diff
@@ -1118,7 +1118,7 @@ func (d *Dispatcher) Session(r *api.SessionRequest, stream api.Dispatcher_Sessio
 		// get the node IP addr
 		addr, err := nodeIPFromContext(stream.Context())
 		if err != nil {
-			log.G(ctx).Debugf(err.Error())
+			log.G(ctx).Debug(err.Error())
```

Comment:
Updated this one to log a descriptive message instead of just spitting out the error.